.github/workflows/ci_test.yml

```diff
@@ -66,3 +66,17 @@ jobs:
         run: |
           set -x -e
           bash tools/run_docker.sh -c 'make sanity-check'
+  public-api-typed:
+    name: Check that the public API is typed
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/setup-python@v1
+        with:
+          python-version: '3.7'
+      - name: install tensorflow and typeguard
+        run: pip install tensorflow-cpu typeguard
+      - name: install tf addons
+        run: TF_ADDONS_NO_BUILD=1 pip install --no-deps -e .
```

Comment:
We should add some information about this env variable in CONTRIBUTING.md

Reply:
Done